test/index.ts

```diff
@@ -2,12 +2,30 @@ global['fetch'] = require('cross-fetch');
 const { JsonRpcProvider } = require('@ethersproject/providers');
 const snapshot = require('../');
 const networks = require('../src/networks.json');
-const example = require('../src/strategies/erc20-balance-of/examples.json')[0];
+
+/* 
+  ## Usage
+  `npm run test` // Tests default (erc20-balance-of)
+  `npm run test --strategy=erc20-received`
+  `npm run test --strategy=eth-balance`
+*/
+
+const strategyNameArgs = Object.keys(snapshot.strategies).map(
+  (name) => `--strategy=${name}`
+);
+const testStrategy =
+  process.argv
```

Comment:
This is super cool! For some reason it always give me "erc20-balance-of" strategy i'm not able to see others strategies when i use `--strategy=erc20-received` argument. When i do a console log of process.env, i just get an array 2 file path but i dont see the argument.

Reply:
Good catch. Yarn/NPM incompatibility. Fixed in https://github.com/snapshot-labs/snapshot.js/pull/60/commits/e81ad16fe8632d7eb752f18df5e7a4527c77aee0 👍